test/unit/server/aws_lambda.server.test.ts

```diff
@@ -76,8 +76,8 @@ describe('Lambda Handler', function () {
                 apiKey: 'testing'
             })
 
-            const handler = client.lambdaHandler(async function(_event, _context) {
-                return mockAwsResult({body:'works!'})
+            const handler = client.lambdaHandler(async function (_event, _context) {
+                return mockAwsResult({body: 'works!'})
```

Comment:
I know this isn't part of this change, but we don't need to always wrap in `mockAwsResult()`. Lambda functions can return anything, and aren't only triggered from API Gateway.

Reply:
You are correct. I think I used this to be consistent in my tests.